frontend/src/metabase/components/ErrorPages/BugReportModal.tsx

```diff
@@ -0,0 +1,120 @@
+import { useState } from "react";
+import { t } from "ttag";
+
+import FormTextArea from "metabase/core/components/FormTextArea";
+import { Form, FormProvider, FormSubmitButton } from "metabase/forms";
+import {
+  Box,
+  Button,
+  Divider,
+  Flex,
+  Loader,
+  Modal,
+  Stack,
+  Text,
+} from "metabase/ui";
+
+import { DiagnosticCheckboxes } from "./DiagnosticCheckboxes";
+import type { ErrorPayload } from "./types";
+
+interface BugReportModalProps {
+  errorInfo: ErrorPayload;
+  onClose: () => void;
+  onSubmit: (values: any) => void;
+  onSlackSubmit: (values: any) => void;
+  canIncludeQueryData: boolean;
+  applicationName: string;
+  hiddenValues: Record<string, boolean>;
+  isSlackSending: boolean;
+}
+
+export const BugReportModal = ({
+  errorInfo,
+  onClose,
+  onSubmit,
+  onSlackSubmit,
+  canIncludeQueryData,
+  applicationName,
+  hiddenValues,
+  isSlackSending,
+}: BugReportModalProps) => {
+  const [isDetailsVisible, setIsDetailsVisible] = useState(false);
+
+  return (
+    <Modal opened onClose={onClose} title={t`Report a bug`} size={550}>
+      <FormProvider
+        initialValues={{
+          ...hiddenValues,
+          queryResults: false,
+          entityInfo: true,
+          frontendErrors: true,
+          backendErrors: true,
+          userLogs: true,
+          logs: true,
+          bugReportDetails: true,
+        }}
+        onSubmit={onSubmit}
+      >
+        {formik => (
+          <Form>
+            <Text py="md">
+              {t`What were you trying to do, and what steps did you take? What was the expected result, and what happened instead?`}
+            </Text>
+            <FormTextArea name="description" autoFocus />
+            <Box bg="#FAFAFB" p="lg" my="md" style={{ borderRadius: "0.5rem" }}>
```

Comment:
I think `#FAFAFB` might be `--mb-base-color-orion-5`. Could we use that variable here?